packages/mui-base/src/Slider/SliderOutput/SliderOutput.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useComponentRenderer } from '../../utils/useComponentRenderer';
+import { useSliderContext } from '../Root/SliderProvider';
+import { sliderStyleHookMapping } from '../Root/styleHooks';
+import { SliderOutputProps } from './SliderOutput.types';
+import { useSliderOutput } from './useSliderOutput';
+
+function defaultRender(props: React.ComponentPropsWithRef<'output'>) {
```

Comment:
You can add `output` to defaultRenderFunctions in utils.

Reply:
👌 Added - initially I didn't because it's such a niche use-case